main/main.ino

```diff
@@ -161,6 +161,8 @@ int failure_number_ntwk = 0; // number of failure connecting to network
 int failure_number_mqtt = 0; // number of failure connecting to MQTT
 #ifdef ZmqttDiscovery
 bool disc = true; // Auto discovery with Home Assistant convention
+#else
+bool disc = false; // Auto discovery with Home Assistant convention
```

Comment:
Not sure you need this if you rebase your change with the development branch.
`disc` should not be used if `ZmqttDiscovery` is commented

Reply:
Removed